docs/readmes/howtos/inbound_roaming.md

```diff
@@ -50,22 +53,27 @@ you need:
 - Functioning [Or8cr](https://docs.magmacore.org/docs/orc8r/architecture_overview),
 - Install [Federatetion Gateway](https://docs.magmacore.org/docs/feg/deploy_intro) and,
 - Install [Access Gateway](https://docs.magmacoreorg/docs/lte/setup_deb).
-- Configure it as a [Federated Deployment](https://docs.magmacore.org/docs/feg/federated_FWA_setup_guide)
+- Create a Federate Deployment (see [below](#Create a Federated Deployment)).
 - Make sure your setup is able to serve calls with your local subscribers
-  (in case of using subscriber DB you will need to set `"hss_relay_enabled":
-  false` on Federated LTE Network temporally to test this. Set it back to
-  `true` once tested!!)
 
-At the end of this you will have:
-- A Federated LTE Network with an LTE Gateway (AGW)
-- A Federation Network with a Federation Gateway (FeG)
+Once you are done you should either:
+- Local Non Federated case:  `a Federated LTE Network with an LTE Gateway (AGW)` + `a Federation Network WITHOUT Federation Gateway (FeG)`
+- Local Federated case: `a Federated LTE Network with an LTE Gateway (AGW)`+
+  `a Federation Network WITH Federation Gateway (FeG)`
 
 We will refer to them as `local` Networks and Gateways to differentiate them
-from the `roaming`Networks and Gateways we will create in the next step.
+from the `roaming` Networks and Gateways we will create in the next step.
+
+### Create a Federated Deployment
+As mentioned, Inbound Roaming requires of a FeG gateway to reach the roaming
+network. That is why Federated Deployment is required. Please, configure it
+using this guide for [Federated Deployment](https://docs.magmacore.org/docs/feg/federated_FWA_setup_guide).
 
-*Note that in case of **Non-Federated + roaming** you can skip the creation
-of the Federated Gateway, but you still need the Federated Gateway Network that will serve your
-Federated LTE Network.*
+In case you don't use local HSS, PCRF and OCS (so Local Non-Federated mode),
+then you will not need to create a FeG Gateway from the previous gide. Also
+remember in your Federated LTE network, you will have to set fields
+`hss_relay_enabled` and `gx_gy_relay_enabled` under `epc` to `false` so request
```

Comment:
don't we need `hss_relay_enabled` to be set to true?

Reply:
ya that is right!